### PR TITLE
fix: Allow user to accept DID method in DID resolver http binding

### LINKED ISSUE #512: Allow user to accept DID method in DID resolver http binding
accept method in DID resolver http binding  always return true

### PR DESCRIPTION
**Pull request by fqutishat**
- Pass accept method in http binding new

closes #512

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/didmethod/httpbinding/resolver.go [hunk 1]
```diff
@@ -86,7 +89,7 @@ func containsDIDDocument(resp *http.Response) bool {
 }
 
 // New creates new DID Resolver
-func New(endpointURL string, opts ...ResolverOpt) (*DIDResolver, error) {
+func New(endpointURL string, acceptDID AcceptDID, opts ...ResolverOpt) (*DIDResolver, error) {
```
Comment (troyronda) [inline pkg/didmethod/httpbinding/resolver.go]: acceptDID isn’t an option?

Comment (fqutishat) [reply pkg/didmethod/httpbinding/resolver.go]: I thought it's mandatory

Comment (troyronda) [reply pkg/didmethod/httpbinding/resolver.go]: unless the default is return true :)

File: pkg/didmethod/httpbinding/resolver.go [hunk 2]
```diff
@@ -20,6 +20,9 @@ import (
 
 var logger = log.New("aries-framework/didmethod/httpbinding")
 
+// AcceptDID is method to accept did doc
+type AcceptDID func(method string) bool
```
Comment (sandrask) [inline pkg/didmethod/httpbinding/resolver.go]: Either Accept or AcceptDIDMethod

File: pkg/didmethod/httpbinding/resolver.go (hunk 1 above)
Comment (fqutishat) [reply pkg/didmethod/httpbinding/resolver.go]: ok I will change to option and default value will be true

Comment (fqutishat) [reply pkg/didmethod/httpbinding/resolver.go]: done

File: pkg/didmethod/httpbinding/resolver.go (hunk 2 above)
Comment (fqutishat) [reply pkg/didmethod/httpbinding/resolver.go]: rename to Accept

Comment (fqutishat) [reply pkg/didmethod/httpbinding/resolver.go]: done

**Pull request merged**